pkg/node/node.go

```diff
@@ -216,22 +225,24 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 		if err != nil {
 			return nil, err
 		}
-	}
 
-	batchStore := batchstore.New(stateStore)
-
-	if !o.Standalone {
-		swapBackend, err := ethclient.Dial(o.SwapEndpoint)
+		erc20Address, err = chequebookFactory.ERC20Address(p2pCtx)
 		if err != nil {
 			return nil, err
 		}
+	}
 
-		chainID, err := swapBackend.ChainID(p2pCtx)
-		if err != nil {
-			logger.Infof("could not connect to backend at %v. A working blockchain node (for goerli network in production) is required. Check your node or specify another node using --swap-endpoint.", o.SwapEndpoint)
-			return nil, fmt.Errorf("could not get chain id from ethereum backend: %w", err)
-		}
+	batchStore := batchstore.New(stateStore)
+
+	post := postage.NewService(stateStore, chainID.Int64())
+
+	// this is needed until postage API gets wired in (since our actual API
+	// falls back to a 32 byte slice of zeros as batch id
+	fallbackBatch := make([]byte, 32)
+	post.Add(postage.NewStampIssuer("empty batch", "", fallbackBatch, 32, 8))
 
+	var postageContractService postagecontract.Interface
+	if !o.Standalone {
```

Comment:
postageStampAddress  is a misnomer below

Reply:
the contract is actually called `PostageStamp`. For now I went with `postageContractAddress`. We should also change the flag which currently is `--postage-stamp-address` but we should reach a decision on naming first as changing that also means changing the helm chart.